src/mixins/validation-mixin.ts

```diff
@@ -5,6 +5,7 @@ import { Component, Vue } from 'vue-facing-decorator'
  */
 @Component({})
 export default class ValidationMixin extends Vue {
+  $v: any
```

Comment:
added as per lint errors

Reply:
I wonder if adding `import { Validations } from 'vuelidate-property-decorators'`, as in BaseAddress.vue, would add "$v" to `this`?

If that doesn't work, I think there's actually a more concise/correct to add something to `this`, but I can't find an example right now. It's related to adding something to the Vue instance, if that helps.

Failing that, if the current code doesn't cause any issues then I'm OK with it.

Reply:
- Adding `import { Validations }...` did not work
- Upgraded vuelidate version
- Initialized `$v` based on documentation

Reply:
Looks great! Do we have any Vue3 components that use the mixins? Do the unit tests pass? Does VS Code show hover help on the mixin methods and no errors?

Reply:
I do not see any component using this but unit tests pass. 

Hover over with no errors:
![image](https://github.com/bcgov/bcrs-shared-components/assets/116035339/cfc1030e-d853-4ee4-8a6c-9fea694e27d8)

Reply:
What if you hover over a mixin method in the unit tests?

Reply:
This is how it looks:
![image](https://github.com/bcgov/bcrs-shared-components/assets/116035339/46ed4de3-8411-4316-a5ca-f8031379b727)

Reply:
Ah, darn. I was hoping the mixin import would have resolved the method signatures. Don't worry about it. The passing unit tests are proof that the mixin methods are present and working.

Reply:
I have no further comments on this conversation.